markdown/developers/modules.md

```diff
@@ -1,1138 +1,177 @@
 ---
-title: DSL2 Modules
-subtitle: Guidelines and reference for DSL2 modules
+title: Module Writing Guidelines
+subtitle: Guidelines on writing a sanger-tol compliant module
 ---
 
-If you decide to upload a module to `nf-core/modules` then this will ensure that it will become available to all nf-core pipelines, and to everyone within the Nextflow community! See [`modules/`](https://github.com/nf-core/modules/tree/master/modules) for examples.
+# NF-Core Module Guidelines
 
-See the [dsl2 modules tutorial](tutorials/dsl2_modules_tutorial) for a step by step guide for how to add a module!
+Generally, modules should conform to the [NF-Core Standards](https://nf-co.re/docs/contributing/modules#new-module-guidelines-and-pr-review-checklist). However, there are a small number of cases where this is not possible.
 
-**⚠️ DSL1 has reached it's end-of-life as of 2023. As of Nextflow version 22.04.x and 22.10.x it will not be possible to run DSL1 scripts. ⚠️**
+----
 
-## Terminology
+# Sanger-tol additions
 
-A domain-specific language (DSL) is a programming language that is developed for a specific application. Nextflow is based on a DSL, where DSL2 is the latest version. DSL2 allows data analysis pipelines to be scaled and modularised. The features offered by Nextflow DSL2 can be used in various ways depending on the granularity with which you would like to write pipelines. Please see the listing below for the hierarchy and associated terminology we have decided to use when referring to DSL2 components.
+## Pipeline Comments (Optional)
```

Comment:
I agree with the content, but I'm not sure `modules.md` is the right file for it.